.gitignore

```diff
@@ -37,7 +37,8 @@ cl/settings/*private.py
 cl/settings/google_auth.json
 .DS_Store
 .dir-locals.el
-
-cl/node_modules
```

Comment:
Wait, shouldn't this be here? Why are you removing it?

Reply:
Not really sure how/why that happened, I certainly didn't do so intentionally. Seems to be fine now though.